src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx

```diff
@@ -0,0 +1,99 @@
+import {Link} from 'react-router';
```

Comment:
this file is a dup of the old settingsBackButton component, minus the marked style block;